sycl/include/sycl/exception.hpp

```diff
@@ -13,11 +13,13 @@
 #include <sycl/detail/cl.h>                   // for cl_int
 #include <sycl/detail/defines_elementary.hpp> // for __SYCL2020_DEPRECATED
 #include <sycl/detail/export.hpp>             // for __SYCL_EXPORT
-#include <sycl/detail/pi.h>                   // for pi_int32
 #include <sycl/detail/string.hpp>
+#include <ur_api.h>                           // for ur_result_t
+#include <ur_print.hpp>                       // to print ur_result_t
```

Comment:
This file is almost 18 000 lines long. We *must not* include it here, compilation time for SYCL headers is already awful as it is. Instead, `stringifyErrorCode` function should be preserved and updated to handle UR error codes instead. This way there will be one less ABI-break.

Reply:
This is also the cause of the e2e slowdown.